haskell/src/Ligo/BaseDAO/Types.hs

```diff
@@ -688,8 +690,8 @@ mkConfig customEps votingPeriod fixedProposalFee maxChangePercent changePercent
       nil
       swap
       dip (push Nothing)
-      constructStack @(DecisionLambdaOutput BigMap)
-  , cCustomEntrypoints = DynamicRec' $ BigMap $ M.fromList customEps
+      constructStack @DecisionLambdaOutput
+  , cCustomEntrypoints = DynamicRec' $ BigMap Nothing $ M.fromList customEps
```

Comment:
Please try to use `mkBigMap <list>` instead of `BigMap Nothing (fromList <list>)` whenever possible.

We have an open ticket to hide the constructor and replace it with more convenient constructor functions: https://gitlab.com/morley-framework/morley/-/issues/625